scvi/data/_built_in_data/_utils.py

```diff
@@ -27,7 +31,18 @@ def read_iter(file, block_size=1000):
     # Create the path to save the data
     if not os.path.exists(save_path):
         os.makedirs(save_path)
+    block_size = 1000
+
+    user_progress_bar_style = scvi.settings.progress_bar_style
 
+    # force progress bar style
+    scvi.settings.progress_bar_style = "tqdm"
```

Comment:
you should explicity pass style into the track function, becaues this will do it globally